src/main/java/uk/gov/companieshouse/web/accounts/links/SmallFullLinkType.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.web.accounts.links;
+
+public enum SmallFullLinkType implements LinkType {
```

Comment:
Do we need to implement an interface here? Is it just for the purpose of unit testing?

Reply:
Nope, just copying what was done in the api :) .. It's handy to have enums as it stops the amount of string literals pasted everywhere in the code further making it easier to change - if a link ever gets changed in the future

Reply:
Fair enough. If that's the case, no need for the `LinkType` interface if the `getLink` method is only implemented here.

Reply:
I think it's best to leave the `LinkType` in as eventually it would be worth adding `CompanyAccountLinkType` etc to extend from it - again following the `company-accounts.api`

Reply:
Okay yep that makes sense to me. Disregard that then.